pkg/coreapi/gql.schema.graphql

```diff
@@ -170,6 +171,73 @@ type FunctionEvent {
   createdAt: Time
 }
 
+type FunctionConfiguration {
+  cancellations: [CancellationConfiguration!]
+  retries: RetryConfiguration!
+  priority: String
```

Comment:
TS SDK shows this type as an object:
https://github.com/inngest/inngest-js/blob/main/packages/inngest/src/components/InngestFunction.ts#L522-L534

Reply:
I noticed that as well but the cloud API is currently using a string https://github.com/inngest/monorepo/blob/dcaef52cdb93e28d0683af80e4a4d59c96f60147/api/schema.graphql#L891

Figured I'd leave it for now until we add more fields to priority unless you think it's worth changing now